frameworks/hdfs/integration/tests/test_utils.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+import dcos
+import shakedown
+
+
+PACKAGE_NAME = 'hdfs'
+WAIT_TIME_IN_SECONDS = 15 * 60
+
+TASK_RUNNING_STATE = 'TASK_RUNNING'
+DEFAULT_HDFS_TASK_COUNT = 8 # 3 data nodes, 3 journal nodes, 2 name nodes
+
+
+def check_health():
+    def fn():
+        try:
+            return shakedown.get_service_tasks(PACKAGE_NAME)
+        except dcos.errors.DCOSHTTPException:
+            return []
+
+    def success_predicate(tasks):
+        running_tasks = [t for t in tasks if t['state'] == TASK_RUNNING_STATE]
+        print('Waiting for {} healthy tasks, got {}/{}'.format(
+            DEFAULT_HDFS_TASK_COUNT, len(running_tasks), len(tasks)))
+        return (
+            len(running_tasks) >= DEFAULT_HDFS_TASK_COUNT,
+            'Service did not become healthy'
+        )
+
+    return spin(fn, success_predicate)
+
+
+def uninstall():
+    print('Uninstalling/janitoring {}'.format(PACKAGE_NAME))
+    try:
+        shakedown.uninstall_package_and_wait(PACKAGE_NAME, app_id=PACKAGE_NAME)
+    except (dcos.errors.DCOSException, ValueError) as e:
+        print('Got exception when uninstalling package, continuing with janitor anyway: {}'.format(e))
+
+    shakedown.run_command_on_master(
+        'docker run mesosphere/janitor /janitor.py '
+        '-r hdfs-role -p hdfs-principal -z dcos-hdfs/hdfs '
```

Comment:
the -z value here should be `dcos-service-hdfs` for the new version of the service